Ethereal Artefacts/Ethereal Artefacts/ContentView.swift

```diff
@@ -9,13 +9,7 @@ import SwiftUI
 
 struct ContentView: View {
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
-        }
-        .padding()
+        LoginView()
```

Comment:
You may move this LoginView in the App file inside the WindowGroup

Reply:
Done.